runners/flink/src/main/java/org/apache/beam/runners/flink/translation/wrappers/streaming/SplittableDoFnOperator.java

```diff
@@ -141,21 +212,182 @@ public void outputWindowedValue(
             sideInputReader,
             executorService,
             10000,
-            Duration.standardSeconds(10)));
+            Duration.standardSeconds(10));
+  }
+
+
+  @Override
+  public void processElement(
+      StreamRecord<WindowedValue<KeyedWorkItem<String, KV<InputT, RestrictionT>>>> streamRecord)
+      throws Exception {
+
+    KeyedWorkItem<String, KV<InputT, RestrictionT>> inputElement =
+        streamRecord.getValue().getValue();
+
+    // Initialize state (element and restriction)
+
+    WindowedValue<KV<InputT, RestrictionT>> windowedValue =
```

Comment:
Hm, I'm not super happy about copy-pasting code of ProcessFn here. Why can't we just use the processFn, like https://github.com/apache/beam/blob/master/runners/direct-java/src/main/java/org/apache/beam/runners/direct/SplittableProcessElementsEvaluatorFactory.java does?